### PR TITLE
Update THIRD-PARTY-LICENSES

### PR DESCRIPTION
**Pull request by thimslugga**
#### Why is this change necessary?

The package name is mispelled.

#### How does it address the issue?

fix mispelling

#### Mandatory Checklist
**PRs will only be reviewed after checklist is complete**

- [ ] Add input/output [type hints](https://docs.python.org/3/library/typing.html) to new functions/methods
- [ ] Write design document if needed ([Do I need to write a design document?](https://github.com/aws/aws-sam-cli/blob/develop/DEVELOPMENT_GUIDE.md#design-document))
- [ ] Write/update unit tests
- [ ] Write/update integration tests
- [ ] Write/update functional tests if needed
- [ ] `make pr` passes
- [ ] `make update-reproducible-reqs` if dependencies were changed
- [ ] Write documentation

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0).

### CODE REVIEW

File: installer/assets/THIRD-PARTY-LICENSES [hunk 1]
```diff
@@ -670,7 +670,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 The above BSD License Applies to all code, even that also covered by Apache 2.0.
 
 --------------------------------------------------------------------------------
-** pygments; version 2.15.0 -- https://pypi.org/project/pygemnts/
+** pygments; version 2.15.0 -- https://pypi.org/project/Pygments/
```
Comment (seshubaws) [inline installer/assets/THIRD-PARTY-LICENSES:673]:
```suggestion
** pygments; version 2.18.0 -- https://pypi.org/project/Pygments/
```

Comment (seshubaws) [reply installer/assets/THIRD-PARTY-LICENSES:673]: The version number should actually be 2.18.0 as seen here https://github.com/aws/aws-sam-cli/blob/d5fd4b803b6a0a9db46ad5102bf0eae8176e635e/requirements/dev.txt#L18

**Comment by seshubaws** [comment]
Hi @thimslugga thank you for opening this PR! I left one comment but otherwise lgtm

**Comment by thimslugga** [comment]
@seshubaws I did a commit to include your suggestion regarding the bump to the version.

**Review by seshubaws — APPROVED** [review]

**Review by lucashuy — APPROVED** [review]

**Pull request merged**